integ-tests/build.gradle

```diff
@@ -17,12 +17,14 @@ dependencies {
     compile project(":data-service-client")
 
     cucumberCompile(
+        'com.google.guava:guava:23.0',
```

Comment:
Doesn't look like anything is using this?

Reply:
multimap